Photon messed up

## Expected Behaviour
To be able to see lighting on other peoples car.

## Actual Behaviour
Cant see lighting on others car (It isnt just that it isnt lighting up, it is that I cant even see the lights on the car

## Error Messages (if any)
[ERROR] addons/photon/lua/autorun/photon/sh_emv_helper.lua:204: attempt to index a nil value
  1. GetIllumSequence - addons/photon/lua/autorun/photon/sh_emv_helper.lua:204
   2. Photon_IllumLights - addons/photon/lua/autorun/photon/cl_emv_meta.lua:126
    3. Photon_RenderIllum - addons/photon/lua/autorun/photon/cl_emv_meta.lua:504
     4. DrawEMVLights - addons/photon/lua/autorun/photon/cl_emv_init.lua:49
      5. v - addons/photon/lua/autorun/photon/cl_emv_init.lua:92
       6. unknown - lua/includes/modules/hook.lua:84

(Unsure if thisis related)

## Steps to Reproduce

  1.  look at someones lights
  2. 
  3. 

## Details
- Source (Workshop / Github):	
- Garry's Mod Beta/Branch (if not standard): tried on 32 and 64

--- a/lua/autorun/photon/sh_emv_helper.lua
+++ b/lua/autorun/photon/sh_emv_helper.lua
@@ -198,7 +198,9 @@ function EMVU.Helper:GetIllumSequence( name, option, vehicle )
 	-- end
 	EMVU.Sequences = EMVU.Sequences or {}
 	if not name or not option or not vehicle then return {} end
-	if not istable( EMVU.Sequences[ name ] ) or not istable( EMVU.Sequences[ name ].Illumination ) then return {} end
+	if not istable(EMVU.Sequences[name]) then return {} end
+	if not istable(EMVU.Sequences[name].Illumination) then return {} end
+	if not istable(EMVU.Sequences[name].Illumination[option]) then return {} end
 	local resultTable = {}
 
 	if IsValid( vehicle ) and istable( EMVU.Sequences[name]["Illumination"][option]["BG_Components"] ) then

